packages/ifpack2/src/Ifpack2_ConfigDefs.hpp

```diff
@@ -77,11 +77,12 @@ namespace Ifpack2 {
     ///   enum may disappear or change name, or the values may change
     ///   names or values or disappear, at any time.
     enum RelaxationType {
-      JACOBI, //!< Jacobi
-      GS,     //!< Gauss-Seidel
-      SGS,    //!< Symmetric Gauss-Seidel
-      MTGS,   //!< Multicore Gauss-Seidel
-      MTSGS   //!< Multicore Symmetric Gauss-Seidel
+      JACOBI,  //!< Jacobi
+      GS,      //!< Gauss-Seidel
+      SGS,     //!< Symmetric Gauss-Seidel
+      MTGS,    //!< Multicore Gauss-Seidel
+      MTSGS,   //!< Multicore Symmetric Gauss-Seidel
+      MTJACOBI //!< Multicore Jacobi
```

Comment:
Is "multithreaded Jacobi" really a different algorithm in exact arithmetic?

Reply:
We need a better name. SplitJacobi ?